ui/src/institutions/components/InstitutionItem.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { Map } from 'immutable';
+import { Row, Col } from 'antd';
+import { LoginOutlined } from '@ant-design/icons';
+
+import ResultItem from '../../common/components/ResultItem';
+import { INSTITUTIONS, LITERATURE } from '../../common/routes';
+import AddressList from '../../common/components/AddressList';
+import WebsitesAction from '../../common/components/WebsitesAction';
+import { getPapersQueryString } from '../utils';
+import IconText from '../../common/components/IconText';
+import ListItemAction from '../../common/components/ListItemAction';
+import InstitutionHierarchyList from './InstitutionHierarchyList';
+
+class InstitutionItem extends Component {
+  render() {
+    const { metadata } = this.props;
+
+    const legacyIcn = metadata.get('legacy_ICN');
+    const recordId = metadata.get('control_number');
+    const addresses = metadata.get('addresses');
+    const urls = metadata.get('urls');
+    const hierarchies = metadata.get('institution_hierarchy');
+
+    return (
+      <ResultItem
+        leftActions={<>{urls && <WebsitesAction websites={urls} />}</>}
+        rightActions={
+          <ListItemAction>
+            <Link to={`${LITERATURE}?q=${getPapersQueryString(recordId)}`}>
+              <IconText text="papers" icon={<LoginOutlined />} />
+            </Link>
+          </ListItemAction>
+        }
+      >
+        <Row type="flex">
```

Comment:
`type="flex"` was removed, and it's by default now AFAIR. (I know that we currently have it in older parts)